Draft

Bala667 opened the pull request:
Hi Ahmed,

I finished updating the GitHub example of machine monitor, Please review and let me know your comments

Bala667 committed:
Add files via upload

Bala667 committed:
Update README.md

overview picture has been updated and Data service name has been changed to IIH essentials

Bala667 committed:
Update README.md

Updated the app versions

Bala667 committed:
Update Installation.md

Changed name and descriptions

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

picture has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Image has been Updated

Bala667 committed:
Update Installation.md

Databus explanation added

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

IIH essentials Pictures has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Configured parameters has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Connectivity troubleshooting has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

monitoring rule has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

First interval rule has been modified

Bala667 committed:
Delete docs/graphics/intervalcreatenewinstruction.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

First rule has been modified

Bala667 committed:
Delete docs/graphics/Monitoringruleoverview.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Second rule picture and description has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Third rule picture and definition has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Asset rules has been modified

Bala667 committed:
Delete docs/graphics/Addruleasset.PNG

Bala667 committed:
Delete docs/graphics/Assignmonitoringrules.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

Assign monitoring rule picture has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Delete docs/graphics/Linkvariable.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Delete docs/graphics/Activate.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Delete docs/graphics/Activate.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Delete docs/graphics/Activate.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Rule activation has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Maintenance rule job has been modified

Bala667 committed:
Delete docs/graphics/Nodue.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Delete docs/graphics/Nodue.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Operation hours is modified

Bala667 committed:
Add files via upload

Bala667 committed:
Delete docs/graphics/variablesoperation.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Maintenance logbook and Event log has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Notification has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Backup and restore has been updated

Bala667 committed:
Add files via upload

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Customize of the machine monitor app has been updated

Bala667 committed:
Update README.md

Bala667 committed:
Update README.md

usage has been updated

Bala667 committed:
Update README.md

Contribution and documentation has been updated , Disclaimer included

ahmedelashy commented on docs/Installation.md:
```diff
@@ -44,11 +44,11 @@ In your IEM open the Databus and launch the configurator.
 `"ie/#"`
 
-![ie_databus_user](graphics/IE_Databus_User.PNG)
+![databususer](graphics/databususer.PNG)
```
Please use a picture where the topic name is seen to avoid confusion.

Bala667 replied:
Hi Ahmed, I changed the picture of Databus

ahmedelashy commented on docs/Installation.md:
```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
+- GDB.process.numberProduced
+- GDB.process.numberFaulty
+
+![S7tags](graphics/S7tags.PNG)
```
Please either change the picture to one without the deploy operation is in progress message or switch this picture with the one after

Bala667 replied:
This changes has been done

ahmedelashy commented on docs/Installation.md:
```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
 Deploy and start the project.
 
-## Configure Data Service
+## Configure IIH Essentials
```
The Configuration of the IIH Essentials part is a part short, would be great if you can explain it a bit more. You can use the old example as a reference

Bala667 replied:
Yeah I added some extra points

ahmedelashy commented on docs/Installation.md:
```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
+
+On the left sidebar, click the `Assets & Connectivity` icon. Under "Hierarchy," you can find the parent asset "edge." To enable the Machine Monitor example to collect data from the IIH Essentials app, you need to create a child asset in the IIH Essentials with a name for the machine.
 
 > Hint: In the current release of the Machine Monitor, it is not possible to collect data from the parent asset "edge."
```
Also this Note is not valid anymore. You can create rules for the parent asset, however for organzitional reasons it is better to have a child asset. Please delete the

Bala667 replied:
I deleted the Note

ahmedelashy commented on docs/Installation.md:
```diff
@@ -58,67 +58,81 @@ In your IEM open the S7 Connector and launch the configurator.
+![Machinemonitoroverview](graphics/Machinemonitoroverview.PNG) 
 
-The settings configured in the Data Service app are imported by the Machine Monitor app. By default, the Machine Monitor starts by opening the parent asset, which in this case is "edge." However, in this example, the child asset "Tank_Filling_Application" is where the assets are configured. The user can click on the child asset, as highlighted in the figure, to preview the available parameters an ...
+The settings configured in the IIH Essentials app are imported by the Machine Monitor app. By default, the Machine Monitor starts by opening the parent asset, which in this case is "edge." However, in this example, the child asset "Tank_Filling_Application" is where the assets are configured. The user can click on the child asset, as highlighted in the figure, to preview the available variables a ...
```
The word settings should be changed with assets.

ahmedelashy commented on docs/Installation.md:
```diff
@@ -127,19 +141,19 @@ The user can now see all the configured data points for this asset, as shown in
-![machine_monitor](graphics/Machine_Monitor_Configured_Datapoints.png)
+![Machinemonitortags.PNG](graphics/Machinemonitortags.PNG)
 
 The Machine Monitor example utilize these acquired data points to create maintenance rules and templates to govern the state and health of the process. The first rule states that when the number of produced bottles exceeds a certain number, a maintenance routine has to be carried out. A second rule can be created for the number of faulty bottles, where if the value of the variable exceeds a certa ...
```
The paragraph should be edited to be a bit more clear. Here is an example:

The Machine Monitor example utilize these acquired data points to create maintenance rules and templates to govern the state and health of the process. In this example four rules will be created to demonstrate the features of the applications. The first rule states that when the number of produced bottles exceeds a certain number, a maintenance routine has to be carried out. A second rule can be created for the number of faulty bottles, where if the value of the variable exceeds a certain threshold, the process has to be stopped and inspected. Third and fourth rules can be created for the temperature and level of the tank to ensure that the process is running correctly and the system is free from any issues.

Bala667 replied:
I updated this description

ahmedelashy commented on docs/Installation.md:
```diff
@@ -127,19 +141,19 @@ The user can now see all the configured data points for this asset, as shown in
+Note: If the configured parameters do not appear in the Machine Monitor application, the user needs to check the configuration of the IIH Essentials application and then verify the credentials used for connecting the Machine Monitor application to the Databus. To do this, click on the "Settings" tab in the sidebar and then select "Connectivity Configuration," as highlighted in the figure below:
 
-![machine_monitor](graphics/Machine_Monitor_Connectivity-Config.png)
+![Machinemonitorconfigsettings](graphics/Machinemonitorconfigsettings.PNG)
```
This picture should be switched with the one after, the order is not correct

Bala667 replied:
I swapped the pictures

ahmedelashy commented on docs/Installation.md:
```diff
@@ -175,44 +189,44 @@ In this example, three rules will be created to demonstrate the features of the
-![machine_monitor](graphics/Machine_Monitor_Interval_Pattern.png)
+![Limitsinstruction](graphics/Limitsinstruction.PNG)
 
 Next, the user can define the "Due limit" and "Overdue limit" for the "Produced Bottles" trigger and set the periodic maintenance period for the conveyor belt. To create new instructions for the maintenance routine, click on "Create New Instruction":
```
We just need to add that the user needs to click on " Interval#1 " to be able to set the values

Bala667 replied:
This changes has been done

ahmedelashy commented on docs/Installation.md:
```diff
@@ -222,39 +236,36 @@ The user needs to define limits for the configured triggers. As highlighted in t
+The three rules are now configured and ready to be associated with the corresponding assets. To assign rules to assets, the user needs to open the "Overview" tab and click on "Add rule" and select "Assign predefined monitoring rules" as highlighted in the figure below:
 
-![machine_monitor](graphics/Machine_Monitor_Asset_Configuration.png)
+![Addruleasset](graphics/Addruleasset.PNG)
```
In assigning rules to assists, the screen shots are taken from the parent asset " edge" and not from the child asset. This should be changed to avoid confusion

Bala667 replied:
I changed the child asset name

ahmedelashy commented on docs/Installation.md:
```diff
@@ -268,114 +279,118 @@ Note: The Machine Monitor app provides the user with a projection for connected
+In order to configure this feature, the user needs to clicks on "Asset settings".
 
-![machine_monitor](graphics/Machine_Monitor_Asset_Config.png)
+![Assetsettings](graphics/Assetsettings.PNG)
```
This first picture is wrong, you meant this one:
![image](https://github.com/industrial-edge/machine-monitor-getting-started/assets/130141248/8559e710-3c6c-4024-9447-782b81738e89)

Bala667 replied:
I highlighted this asset settings

ahmedelashy reviewed (commented):
Hi Bala, I reviewed the example. Here you should find my comments. Thanks in advance

Bala667 committed:
Update Installation.md

Databus user picture has been modified

Bala667 committed:
Delete docs/graphics/Databususerpreview.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

S7 connector config picture has changed

Bala667 committed:
Delete docs/graphics/S7tags.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

IIH Simatic S7 connector has been modified

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

IIH essential additional step has been added and SIMATIC S7 connector has been changed

Bala667 committed:
Update Installation.md

Note has been deleted

Bala667 committed:
Update Installation.md

The word settings has been changed to assets

Bala667 committed:
Update Installation.md

Configure Machine Monitor description has been changed

Bala667 committed:
Update Installation.md

The pictures has been switched for Connectivity troubleshooting

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

Bala667 committed:
Delete docs/graphics/Assetsettings.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Child asset name has been changed

Bala667 commented:
Hi Ahmed, I modified the GitHub example according to your review.

ahmedelashy reviewed (commented):
For the  **Configure an asset with variables** part ,add those steps please to avoid confusion:

![image](https://github.com/industrial-edge/machine-monitor-getting-started/assets/130141248/e3e9383c-d076-4534-b9bd-d55cc1c48e14)

![image](https://github.com/industrial-edge/machine-monitor-getting-started/assets/130141248/2c020df8-dd52-4837-a2de-14c89f607240)


For the  **Machine Monitor maintenance logback and event log:**:

The same comment as before, please either use the Tank_filling_Application as an asset or the edge one as an asset. If you use both in the pics it will cause confusion

![image](https://github.com/industrial-edge/machine-monitor-getting-started/assets/130141248/78c4e0b0-7abe-4ab2-b7aa-ea172390aa53)

Bala667 committed:
Update Installation.md

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

IIH essentials description got elaborated

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Screen shot changed to asset with Tank filling Application

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md

Bala667 committed:
Delete docs/Lubricationrule.PNG

Bala667 committed:
Delete docs/Linkvariable.PNG

Bala667 committed:
Delete docs/Assignmonitorruletank.PNG

Bala667 committed:
Add files via upload

Bala667 committed:
Add files via upload

Bala667 committed:
Update Installation.md
